docs/pod-metrics.md

```diff
@@ -36,7 +36,7 @@
 | kube_pod_deletion_timestamp                           | Gauge       | Unix deletion timestamp                                                                                                                                                             | seconds                                        | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `uid`=&lt;pod-uid&gt;                                                                                                                                                                                                                                                                          | EXPERIMENTAL | -      |
 | kube_pod_restart_policy                               | Gauge       | Describes the restart policy in use by this pod                                                                                                                                     |                                                | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `type`=&lt;Always\|Never\|OnFailure&gt; <br> `uid`=&lt;pod-uid&gt;                                                                                                                                                                                                                             | STABLE       | -      |
-| kube_pod_init_container_info                          | Gauge       | Information about an init container in a pod                                                                                                                                        |                                                | `container`=&lt;container-name&gt; <br> `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `image`=&lt;image-name&gt; <br> `image_id`=&lt;image-id&gt; <br> `image_spec`=&lt;image-spec&gt; <br> `container_id`=&lt;containerid&gt; <br> `uid`=&lt;pod-uid&gt;                                                                                    | STABLE       | -      |
+| kube_pod_init_container_info                          | Gauge       | Information about an init container in a pod                                                                                                                                        |                                                | `container`=&lt;container-name&gt; <br> `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `image`=&lt;image-name&gt; <br> `image_id`=&lt;image-id&gt; <br> `image_spec`=&lt;image-spec&gt; <br> `container_id`=&lt;containerid&gt; <br> `uid`=&lt;pod-uid&gt; <br> `type`=&lt;Always&gt;                                                                                    | STABLE       | -      |
```

Comment:
```suggestion
| kube_pod_init_container_info                          | Gauge       | Information about an init container in a pod                                                                                                                                        |                                                | `container`=&lt;container-name&gt; <br> `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `image`=&lt;image-name&gt; <br> `image_id`=&lt;image-id&gt; <br> `image_spec`=&lt;image-spec&gt; <br> `container_id`=&lt;containerid&gt; <br> `uid`=&lt;pod-uid&gt; <br> `type`=&lt;restart-policy&gt;                                                                                    | STABLE       | -      |
```

One last change.

Reply:
`<br> `restart_policy`=&lt;Always&gt;` right?

Reply:
The `Always` should be replaced by a placeholder

Reply:
@mrueg 
i fixed